tests/e2e/core-tests/specs/merchant/wp-admin-order-searching.test.js

```diff
@@ -65,39 +60,33 @@ const updateCustomerBilling = async () => {
 		shipping: customerShipping,
 	};
 	await client.put( customerEndpoint + customerId, customerData );
+	return customerId;
 };
 
 const runOrderSearchingTest = () => {
 	describe('WooCommerce Orders > Search orders', () => {
+		let productId;
 		let orderId;
-		beforeAll( async () => {
-			await createSimpleProduct('Wanted Product');
-			await updateCustomerBilling();
+		let customerId;
 
-			// Create new order for testing
+		beforeAll( async () => {
+			productId = await createSimpleProduct('Wanted Product');
+			customerId = await updateCustomerBilling();
+			orderId = await createOrder({
+				customerId: customerId,
```

Comment:
These can all be simplified to just the variable name.

Reply:
Done! 😄